Allure.Reqnroll/Events/TestOutputEventHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Allure.Net.Commons;
+using Allure.ReqnrollPlugin.State;
+using Reqnroll.Events;
+
+namespace Allure.ReqnrollPlugin.Events;
+
+class TestOutputEventHandler : AllureReqnrollEventHandler<OutputAddedEvent>
+{
+    public TestOutputEventHandler(
+        Func<CrossBindingContextTransport> stateStorageFactory
+    ) : base(stateStorageFactory)
+    {
+    }
+
+    protected override void HandleInAllureContext(OutputAddedEvent eventData) {
+        AllureReqnrollStateFacade.AddOutput(
+            eventData.Text + "\n"
```

Comment:
Since this is the only place where `AddOutput` is called, we may reduce the number of concatenations by using `StringBuilder`'s `AppendLine` instead of appending the new line character here and calling `Append`.